newton/_src/solvers/vbd/rigid_vbd_kernels.py

```
@@ -0,0 +1,2445 @@
+    joint_penalty_k[i] = k_new
+
+
+@wp.kernel
+def warmstart_body_body_contacts(
+    rigid_contact_count: wp.array(dtype=int),
+    rigid_contact_shape0: wp.array(dtype=int),
+    rigid_contact_shape1: wp.array(dtype=int),
+    shape_material_ke: wp.array(dtype=float),
+    shape_material_kd: wp.array(dtype=float),
+    shape_material_mu: wp.array(dtype=float),
+    gamma: float,
```

Ruff (0.14.6)

Unused function argument: `gamma`

(ARG001)